src/sentry/static/sentry/app/views/settings/incidentRules/types.tsx

```diff
@@ -118,6 +118,14 @@ export type MetricActionTemplate = {
    * Integration id for this `type`, should be passed to backend as `integrationId` when creating an action
    */
   integrationId: number;
+  /**
+   * The input type on the front end, e.g. text for Slack, select for PagerDuty
+   */
+  inputType: string | null;
```

Comment:
@ceorourke Probably better to have `inputType?: string;` than to explicitly allow nulls.